types/chart.js/index.d.ts

```diff
@@ -396,6 +396,7 @@ declare namespace Chart {
     interface TickOptions {
         autoSkip?: boolean;
         autoSkipPadding?: number;
+        beginAtZero?: boolean;
```

Comment:
Could you remove this setting from LinearTickOptions a few lines below, to avoid the duplication?

Reply:
Sure can!

Reply:
Did you want me to move those other ones as well?

Reply:
Moving just the stuff that particularly concerns you is fine, but if you have the time to move everything else, that'd be great! However, the interface must be kept (even if empty), in case someone is currently using it.

Reply:
Is there any reason why there are two TickOptions interfaces?

Reply:
I don't think so. I bet whoever wrote the second one just didn't notice what was there before.